index.html

```diff
@@ -21,15 +21,11 @@
         </div>
         <section class="headline-section">
```

Comment:
- **[OPTIONAL]** Please make sure your website has the title that best describes what your portfolio stands for 👊🏽.

![image](https://user-images.githubusercontent.com/43172164/200028328-bb86865e-2940-4268-a4c4-481566498615.png)

Reply:
@Yazino12 done

Reply:
@Yazino12 DONE

Reply:
@Yazino12 Please review